packages/lesmis/src/components/Layout.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+
+import styled from 'styled-components';
+import MuiContainer from '@material-ui/core/Container';
+import MuiBox from '@material-ui/core/Box';
+
+const desktopWidth = '768px';
+
+/*
+ * Wrapping container
+ */
+export const Container = styled(MuiContainer)`
```

Comment:
No need to review this file. The layout is coming later on card: https://github.com/beyondessential/tupaia-backlog/issues/2279